frame/fast-unstake/src/benchmarking.rs

```diff
@@ -43,10 +43,14 @@ fn l<T: Config>(
 	T::Lookup::unlookup(who)
 }
 
-fn create_unexposed_nominator<T: Config>() -> T::AccountId {
-	let account = frame_benchmarking::account::<T::AccountId>("nominator_42", 0, USER_SEED);
-	fund_and_bond_account::<T>(&account);
-	account
+fn create_unexposed_nominators<T: Config>() -> Vec<T::AccountId> {
+	(0..T::BatchSize::get())
+		.map(|i| {
+			let account = frame_benchmarking::account::<T::AccountId>("nominator_42", i, USER_SEED);
```

Comment:
should this be “nominator_$i”?

Reply:
the next argument `i` is already separating them, will just remove the 42 here.